readme/Configuration.scalatex

```diff
@@ -381,6 +381,26 @@
       implicit def validatedInstances[E](implicit E: Semigroup[E]): Traverse[
           Validated[E, ?]] with ApplicativeError[Validated[E, ?], E] = 2
 
+  @sect{newlines.beforeCaseClassParameterDefn}
+    Together with @b{verticalMultilineAtDefinitionSite}, this will
+    align parameter definitions vertically, with a line for each, and
+    not attempt to fit the case class definition on a single line.
+
+    Default @b(default.newlines.beforeCaseClassParameterDefn)
+
+    Requires @b{verticalMultilineAtDefinitionSite} set to @b(true).
+
+    @hl.scala
+      // verticalMultilineAtDefinitionSite = true
+      // newlines.beforeCaseClassParameterDefn = false
+      case class A(b: Int, c: Int)
```

Comment:
Why only case classes and not regular classes?

Reply:
Good question. I probably had our use case a bit too in mind. :-)

Would it be better with a multi-option setting?

Reply:
Ping? :-)

Reply:
I apologize the slow response, this got totally lost in my inbox.

I am personally not a fan of this option but I can understand the motivation. I'm OK with adding this option, but I'll leave it to @pjrt to LGTM the changes since he is the author of `verticalMultilineAtDefinitionSite`.